lib/purescript-ast/package.yaml

```diff
@@ -0,0 +1,61 @@
+name: purescript-ast
+version: '0.13.6'
+synopsis: PureScript Programming Language Abstract Syntax Tree
+description: Defines the underlying syntax of the PureScript Programming Language.
```

Comment:
Wasn't entirely sure what to put here. Feel free to suggest any changes if you'd like.